tests/mesh/boundary_info.C

```diff
@@ -20,41 +21,35 @@ public:
   CPPUNIT_TEST_SUITE( BoundaryInfoTest );
 
   CPPUNIT_TEST( testMesh );
+  CPPUNIT_TEST( testEdgeBoundaryConditions );
 
   CPPUNIT_TEST_SUITE_END();
 
 protected:
 
-  Mesh* _mesh;
-
-  void build_mesh()
-  {
-    _mesh = new Mesh(*TestCommWorld);
-
-    MeshTools::Generation::build_square(*_mesh,
-                                        2, 2,
-                                        0., 1.,
-                                        0., 1.,
-                                        QUAD4);
-  }
-
 public:
   void setUp()
   {
-    this->build_mesh();
```

Comment:
@roystgnr Is there some reason that we `new`/`delete` meshes in these unit tests?  It looks like an idiom that has been copy/pasted around for a while, but I can't remember if there was a reason for it...